README.rst

```diff
@@ -399,6 +399,60 @@ Now, checking again gives ``adrian`` the required permissions:
     False
 
 
+Tieing permissions to models
```

Comment:
I would use "Permissions in models" as the heading, to be somewhat consistent with the other headings in this section.

Reply:
Hmm, how about change all headings like "Permissions in models"? I find it a little confusing to read "Rules and permissions", because this is all about the permissions part of `rules` only, and not the more generic rules.

Reply:
Just did it and think it looks good, but we can of course do it the other way if you prefer.

Reply:
No, that's fine -- I didn't realise we only discuss permissions in these sections, except the templates (which was done first and works with rules, not only permissions) and it's where I probably copied the section titles from.

Reply:
Ok, "Permissions and rules in templates" now is the only heading including "rules".